org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -264,6 +264,51 @@ GithubOidcCfnTemplateDeploy:
     Account: !Ref AdminCentralAccount
     Region: us-east-1
 
+# GH OIDC for lambda template deployments to AWS AdminCentral account
+GithubOidcLambdaTemplateDeploy:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+  StackName: !Sub ${resourcePrefix}-${appName}-lambda-template-deploy
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-lambda-template-deploy
+    PolicyDocument: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+              "Sid": "ListObjectsInBucket",
+              "Effect": "Allow",
+              "Action": [ "s3:GetBucketLocation", "s3:ListBucket", "s3:ListBucketV2",
+                          "s3:ListParts", "s3:ListMultipartUploads"
+                        ],
+              "Resource": [ "arn:aws:s3:::bootstrap-awss3cloudformationbucket-19qromfd235z9",
+                            "arn:aws:s3:::essentials-awss3lambdaartifactsbucket-x29ftznj6pqw"
+                          ]
+          },
+          {
+              "Sid": "AllObjectActions",
+              "Effect": "Allow",
+              "Action": [ "s3:PutObject", "s3:GetObject", "s3:DeleteObject", "s3:DeleteObjects",
+                          "s3:UploadPart", "s3:CompleteMultipartUpload", "s3:AbortMultipartUpload",
+                          "s3:CreateMultipartUpload"
+                        ],
+              "Resource": [ "arn:aws:s3:::bootstrap-awss3cloudformationbucket-19qromfd235z9/*",
+                            "arn:aws:s3:::essentials-awss3lambdaartifactsbucket-x29ftznj6pqw/*"
+                          ]
+          }
+        ]
+      }
+  TemplatingContext:
+    GitHubOrg: "Sage-Bionetworks-IT"
+    Repositories:
+      - name: "lambda-*"
```

Comment:
we have lambdas that do not exclusively match this.  it will not cover `Sage-Bionetworks-IT/lambda-template`, `aws-lambfda-efs-samples`, `s3-synapse-sync`, `cfn-cr-alb-rule`, etc..

Reply:
sorry, `aws-lambda-efs-samples` isn't one of our repos.  it's a fork of an AWS labs repo.